automated/linux/pi-stress/pi-stress.sh

```diff
@@ -53,20 +53,13 @@ fi
 
 background_process_start bgcmd --cmd "${BACKGROUND_CMD}"
 
-# pi_stress will send SIGTERM when test fails. The single will terminate the
+# pi_stress will send SIGTERM when test fails. The signal will terminate the
 # test script. Catch and ignore it with trap.
 trap '' TERM
-"${binary}" --duration "${DURATION}" "${MLOCKALL}" "${RR}" | tee "${LOGFILE}"
+"${binary}" -q --duration "${DURATION}" ${MLOCKALL} ${RR} --json="${LOGFILE}"
```

Comment:
Shouldn't ${MLOCKALL} ${RR} be escaped?

Reply:
Yes, let me update it.

Reply:
Ah no wait. The very purpose of this commit is to drop the quotes. I tried to express this in the commit message but obviously failed.  When the user sets MLOCKALL to false (or RR to false) the resulting command line contains a two apostrophe which get translated into an empty argv entry. This is makes the command line parser stop processing the command line and hence we will miss any command line option after the first empty quote. Just to illustrate it, in case you
set MLOCKALL=false and RR=true, the RR=true would never be picked up.

Since we have two command line option which could be set to false, we can't just move everything non 'false' before the last command line option. I am no bash wizard, so there might be another way to fix this problem. I just figured dropping
the quotes makes the parser work.

Reply:
make sense, thank you for the explanation